viper.go

```diff
@@ -1177,6 +1181,24 @@ func readAsCSV(val string) ([]string, error) {
 	return csvReader.Read()
 }
 
+func parseStringToStringFlagValue(val string) map[string]interface{} {
```

Comment:
How about copying the same function from pflag and keeping the implementation as close and possible?

https://github.com/spf13/pflag/blob/master/string_to_string.go#L79

Reply:
@sagikazarmark i've made this change. i had to tweak the signature and return types slightly to work with GetStringMap that uses cast.ToStringMap

Reply:
Sure thing, thanks!